### DISCUSSION TITLE
Fatjets

### DISCUSSION THREAD

**Discussion by fnesti** [MadAnalysis/madanalysis5#76] [Q&A] [answered]
Hi, maybe simple question: we see madanalysis has interface for the delphes FatJet branch.  But is there a way to access it in normal RECO mode?  Or are fatjets included in "j" ?

ma5>plot PT(
a          e          e-         invisible  l          l-         mht        mu+        mu-        mu_isol    ta         ta-
b          e+         hadronic   j          l+         met        mu         mu+_isol   mu-_isol   nb         ta+        

Thanks in advance, and thanks for the great work

**Comment by jackaraz** [comment, marked as answer]
Hi @fnesti, fat jets are only available through expert mode at the moment.

**Reply by fnesti** [reply]
Thanks, I see.
